components/public-api/gitpod/v1/configuration.proto

```diff
@@ -19,12 +19,20 @@ message Configuration {
   WorkspaceSettings workspace_settings = 7;
 }
 
+enum PrebuildActivationStrategy {
```

Comment:
nit: It's not the startegy for "activating" prebuilds, but for triggering prebuilds.